### DISCUSSION TITLE
Is the nuxt.com website code open-sourced ?

### DISCUSSION THREAD

**Discussion by abhiram11** [nuxt/nuxt#19255] [Questions]
Hi,

I was amazed by how beautifully the `nuxt.com` website has been developed. However, I came across something in the UI that I thought could be improved, and its something that I can contribute to.

However, I was not able to find the source code for the official website `nuxt.com` anywhere in any repository. 🙁 I searched for other discussions and issues such as #16235 but did not find my answer up to the best of my knowledge.

I would really appreciate it if you could direct me to the code repo if it is available for contribution.

Thanks in Advance!

**Comment by abdul-alhasany** [comment]
Hi
I think it is not open sourced (yet).

Currently you can contribute only to the content. See here: https://github.com/nuxt/nuxt/tree/main/docs

**Reply by abhiram11** [reply]
That's a bummer, but thank you for the answer! I'll be waiting for it to be open source then!
